lib/peep.rb

```diff
@@ -0,0 +1,32 @@
+require 'pg'
+
+class Peep
+
+  attr_reader :id, :message, :peep_date
+
+  def initialize(id, message, peep_date)
+    @id = id
+    @message = message
+    @peep_date = peep_date
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
```

Comment:
It's worth considering that this if/else statement is repeated below as well, to follow the DRY principle, you could, for example, create a function that takes the `environment` information and then reuse it to establish the connection with the test or the main database.